vulyk/control.py

```diff
@@ -115,21 +119,40 @@ def group_resign_to(username, gid):
     _groups.resign(username, gid)
 
 
+@group.command("addtype")
 @click.option("--gid",
               prompt="Specify group's id",
               type=click.Choice(_groups.get_groups_ids()))
-@group.command("addtype")
 @click.option("--task_type",
+              type=click.Choice(TASKS_TYPES.keys()),
               prompt="Provide the task type name")
 def group_addtype(gid, task_type):
     _groups.add_task_type(gid, task_type=task_type)
 
 
+@group.command("deltype")
 @click.option("--gid",
               prompt="Specify group's id",
               type=click.Choice(_groups.get_groups_ids()))
-@group.command("deltype")
 @click.option("--task_type",
+              type=click.Choice(TASKS_TYPES.keys()),
               prompt="Provide the task type name")
 def group_deltype(gid, task_type):
     _groups.remove_task_type(gid, task_type=task_type)
+
+
+@cli.command("init")
+@click.argument("allowed_types",
+                type=click.Choice(TASKS_TYPES.keys()),
+                nargs=-1)
+def project_init(allowed_types):
+    """
+    Bootstrapping
+
+    :type allowed_types: list[basestring]
+    """
+    if len(allowed_types) == 0:
+        raise click.BadParameter("Please specify at least "
+                                 "one default task type")
+
+    _project_init(allowed_types)
```

Comment:
Let's add some auxiliary command to print out all groups with assigned types and also the list of all task types